_overviews/scala3-contribution/contribution-intro.md

```diff
@@ -4,42 +4,20 @@ type: chapter
 description: This page describes the format of the contribution guide for the Scala 3 compiler.
 num: 1
 previous-page:
-next-page: start-intro
+next-page:
```

Comment:
Could you please confirm that the rendered sidebar does not show the other pages anymore?

Reply:
I’ve just built the site locally and I confirm that there is still a sidebar (which should be removed):

![image](https://github.com/scala/docs.scala-lang/assets/332812/fc448527-0afb-41ee-b60d-8b41bed882d4)

Maybe this can be fixed by changing the page layout?

Also, I see that the page is listed in the “Guides & Overviews”:

![image](https://github.com/scala/docs.scala-lang/assets/332812/2bfc1025-8183-4ba4-86c6-6dfca5b5e1e9)

I think it should be removed from here and instead added as a page there: https://docs.scala-lang.org/contribute/

I can help you if needed to preserve the old URLs… Just let me know!

Reply:
Hello Julien. Yes sorry, I am currently changing locally but strangely the command I run "bundle exec jekyll serve -I --watch" instantly updates the redirect links but not the display (which takes a really long time). For example for the following code from the `procedures-cheatsheet.md` page.

<img width="1080" alt="Screenshot 2023-05-17 at 15 31 31" src="https://github.com/scala/docs.scala-lang/assets/44496264/2855be1e-e87d-46f4-bb58-532610da665d">

I have the `redirect_to: https://dotty.epfl.ch/docs/contributing/workflow/index.html#sbt-commands-cheat-sheet-1ddada` redirecting nicely to this URL in my tests (For a test).
However, the name displayed remains `Cheatsheetsdasdsa` in the display (An old name I had for testing before yesterday).

<img width="1303" alt="Screenshot 2023-05-17 at 15 32 15" src="https://github.com/scala/docs.scala-lang/assets/44496264/bf7414af-2580-417c-aa77-2636ac14bf26">

So I think I have the solution to remove the pages from the sidebar (Remove the previous/next page and/or set the type to simple). But it takes a really long time before I can see the result. I'm not sure if it's the command or something else (I've cleared the cache, etc.)

Reply:
Hey, I’ve also noticed that the incremental compilation in Jekyll does not work well sometimes… I couldn’t find a better solution than removing files to force their re-generation. Instead of removing all the website, I only remove the files that I want to re-generate (in `_site`), I think that makes it slightly faster…

Reply:
But anyway, I think that page should not be in the “Guides & Overviews” but in the existing page “Become an OSS Contributor”, and in that case there will be no sidebar anymore.

Reply:
Ho okay super ! Thank you very much for the tip. I'll send the correction in a few minutes then !

Reply:
PR sent and desc updated @julienrf. So I did a `redirect-to` to redirect to the "Contribute to Scala3" in `Contributing to scala's oss` but I can also just change the link in `/_overviews/contribute/guide.md` if it's better.